### PR TITLE
Resistor Color approach

### PR DESCRIPTION
**Pull request by bobahop**
An approach for Resistor Color.

### CODE REVIEW

**Comment by bobahop** [comment]
@wolf99 and/or @ErikSchierboom: C approach for Resistor Color ready for review.

**Comment by siebenschlaefer** [comment]
Two small things:

- The initial `.h` file includes `<stdint.h>` and `TEST_ASSERT_EQUAL_UINT16` in the tests explicitly expects `color_code()` to return a `uint16_t`. Do you mind changing the return type from `int` to `uint16_t`?
- A function declaration like `colors()` with empty parentheses is valid C but it's actually a remnant of KR-style C that didn't declare the number or types of the parameter. Do you mind changing it to `colors(void)`?

File: exercises/practice/resistor-color/.approaches/introduction.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Introduction
+
+There are several ways to solve Resistor Color.
+One approach is to use the [`#define`][define] directive to set the values needed for both the `enum`  and the array.
+
+# General guidance
```
Comment (wolf99) [inline exercises/practice/resistor-color/.approaches/introduction.md:6]:
```suggestion
## General guidance
```
Should only be one top level heading per md document.

**Review by wolf99 — CHANGES_REQUESTED** [review]

**Comment by wolf99** [comment]
+1 to @siebenschlaefer comments

**Comment by bobahop** [comment]
Thanks for the catches! All changes implemented.

**Review by wolf99 — APPROVED** [review]

**Pull request merged**